.gitignore

```diff
@@ -158,3 +158,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+.DS_Store
+fmatch/main.py
```

Comment:
Is main.py a local development tool?

Reply:
Yes Joe, it was intended to test the functions in my local.